clang/test/Driver/sycl.c

```diff
@@ -22,15 +22,19 @@
 // DEFAULT-PHASES-NOT: linker
 
 // -fsycl-help tests
+// RUN: mkdir -p %t-sycl-dir
```

Comment:
shouldn't we do the same WA for ioc64 and ocloc because in general these tools might be missed on on machine where check-clang tests are executed.